### ISSUE TITLE
backup / recovery flows

### ISSUE THREAD

**Issue by johncantrell97** [L2-Technology/sensei#9]
currently there is no backup or recovery flows.  need to review and brainstorm how this should work.  I'd like to support multiple options for backup (filesystem, cloud (google drive / dropbox / etc), photon?, synonum backpack?).

before supporting  multiple backup options it probably makes sense to handle recovery.

I like what bluewallet has done where they encrypt the backup with your mnemonic so that you can recover your backup and on-chain funds with a single mnemonic.

**Comment by xraid** [comment]
as a first rudimentary recipe backup one could use rsync

have a interval or watch encrypt to a backupthis-folder that in turn rsync to a remote

i se this as a parallel running micro-service rust-app that has watch and trigger

**Comment by johncantrell97** [comment]
yeah that could be an option.  most of these issues are just placeholders with ideas not fully fleshed out. 

we don't even currently expose the mnemonic used, that's probably the easiest first step.
from there a recovery flow would be a good next step.
finally adding some alternative encrypted backup options

**Comment by xraid** [comment]
HelloLightning (BlueWallet kotlin LDK Node) writes mnemonic seed to disk, i find that good altho maybe extend with a .env variable PIN ie.

**Comment by xraid** [comment]
HelloLightning (BlueWallet kotlin LDK Node) writes mnemonic seed to disk, i find that good !?,  maybe extend with a .env variable PIN ie.

export MENEMONIC_PIN=xxxxxx

and have system encrypt mnemonic to disk where also with the help of script be able decode mnemonic.disk using PIN
